_posts/2024-08-23-GSoC-CSL.md

```diff
@@ -0,0 +1,65 @@
+As a student, I first came across pre-defined citation formats during my freshman year of college, when our professor instructed that the references in our project report must follow the "APA style".  
+
+I remember manually generating and copying citations in that style from some citation generation website, and oh was it tedious. Whether you are a student like me or a seasoned researcher in academia, you must have grappled with various citation styles throughout your academic journey.
+
+[Citation Style Language](https://citationstyles.org/) (CSL) is a popular open-source specification language that standardizes the formatting of citations and bibliographies. It supports thousands of standard citation styles used in academia, including *American Psychological Association* (APA), *Modern Language Association* (MLA), *Chicago Manual of Style* (CMS), *Institute of Electrical and Electronics Engineers* (IEEE), *Springer - Lecture Notes in Computer Science* and *Vancouver* to name a few. If you've ever cited a source in an academic paper, chances are you've used a CSL-based style.
+
+For several years, JabRef users have been requesting support for CSL in its LibreOffice integration. Tackling this problem statement this summer as a part of my project, I am excited to announce that this highly anticipated feature is now available!
+
+## What's New?
+
+With this integration, JabRef users can now:
+
+1. **Choose a CSL Style**: Select from a vast array of CSL styles to match the requirements of your academic work.
+2. **Cite with Flexibility**: Insert citations as well as in-text citations (similar to LaTeX's `\cite` and `\citet` commands) directly into your LibreOffice document.
+3. **Auto-generate Bibliographies**: Watch as your bibliography automatically populates based on your citations, with just a single click.
+4. **Update Bibliographies**: Easily refresh your bibliography as you add new citations to your document.
+
+## Getting Started
+
+To start using this new feature:
+
+1. Update to the latest version and start JabRef
+2. Connect to a running LibreOffice document instance from the OpenOffice Panel
+3. Click on "Select style" from the panel
+4. Choose your preferred CSL style from the list and click OK
+5. Select an entry (or a group of entries) and click "Cite" to insert citations into the document
+6. Click on the "Make/Sync bibliography" on the panel to auto-generate a bibliography section in your document based on your citations and the selected CSL style
+
+### Demo
+
+#### Selecting a style
+
+![Selecting a style](../img/selecting-a-style.gif)
+
+#### Inserting citations
+
+![Inserting citations](../img/cite.gif)
+
+#### Generating bibliography
+
+![Generating bibliography](../img/bibliography.gif)
+
+#### Updating bibliography
+
+![Updating bibliography](../img/refresh.gif)
+
+## Summary
+
+We hope this new feature enhances your research and writing process. As always, we welcome your feedback and suggestions for further improvements.
```

Comment:
Add link to forum for feedback - otherwise, users might not know where to put feedback.

```suggestion
I hope this new feature enhances your research and writing process. As always, [we welcome your feedback and suggestions for further improvements](https://discourse.jabref.org/c/feedback/3).
```

Reply:
Also move the paragraph as last.

Reason: Structure of a summary is:

1. Summary
2. Future work